### DISCUSSION TITLE
Option to set axis' manually on armature import

### DISCUSSION THREAD

**Discussion by TackYs** [niftools/blender_niftools_addon#525] [Ideas]
When importing armatures, axis are currently set automatically by the guess_orientation function in nif_import/armature/init.py:

```
# use heuristics to determine a suitable orientation 
forward, up = self.guess_orientation(n_armature) 
# pass them to the matrix utility 
math.set_bone_orientation(forward, up)
```

Some times it would be valuable to be able to set these manually, example is if you have a rigged armature in the scene with a specific orientation and want to import another armature with matching orientation. 

I'd love to give this a shot and pull request, is this a reasonable feature though?

Wrote this on mobile, hope formatting comes out ok.

**Comment by neomonkeus** [comment]
You would need to update the UI 
- to have some form of override toggle. 
- provide the user with the up, forward values.

Then update the function shown above to, before doing the guess check 
- if the override is set, 
- if set, set forward, up to the values provided by the user
